snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -119,11 +121,29 @@ open class Snapshot: NSObject {
     }
 
     class func pathPrefix() -> NSString? {
-        if let path = ProcessInfo().environment["SIMULATOR_HOST_HOME"] as NSString? {
-            return path.appendingPathComponent("Library/Caches/tools.fastlane") as NSString?
-        }
-        print("Couldn't find Snapshot configuration files at ~/Library/Caches/tools.fastlane")
-        return nil
+        var homeDir: NSString
```

Comment:
Not sure why we use an `NSString` here and then cast it down below in the return statement?